app/controllers/kudos_controller.rb

```diff
@@ -66,6 +67,34 @@ def create
 
     if kudo.save
       render json: { kudo: kudo }, status: :created
+
+      # Success, send email notification
+      ReceivedKudosMailer.received_kudos_notification(kudo).deliver_now
```

Comment:
Was this PR built off of master or the email branch? It looks like it was built off of the email branch — that'll probably lead to some headaches/conflicts when merging back into master.